tests/src/system/basic/WskBasicTests.scala

```diff
@@ -466,6 +466,19 @@ class WskBasicTests
             wsk.trigger.list().stdout should include(name)
     }
 
+    it should "create a trigger using property file" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "listTriggers"
+            val tmpProps = File.createTempFile("wskprops", ".tmp")
+            val env = Map("WSK_CONFIG_FILE" -> tmpProps.getAbsolutePath())
+            wsk.cli(Seq("property", "set", "--auth", wp.authKey) ++ wskprops.overrides, env = env)
```

Comment:
`wskprops.overrides` are intended to precede the sub command whereas in the above they come after - is this intentions? note this is what happens in this case: `wsk property set --auth xxx --apihost yyy --auth xxx` since `auth` is one of the overrides.